src/reducers/secrets.js

```diff
@@ -77,11 +77,11 @@ function isFetching(state = false, action) {
   }
 }
 
-function errorMessage(state = null, action) {
+function errorMessage(state = false, action) {
   switch (action.type) {
     case 'SECRETS_FETCH_FAILURE':
     case 'SECRET_CREATE_FAILURE':
-      return action.error;
+      return 'The server reported a conflict';
```

Comment:
This doesn't seem right, in the case of `FETCH_FAILURE` why would this be caused by a conflict? Could the create also fail for other reasons?

Reply:
Yep.... think I changed it during testing just to make sure the right error was being got from the reducer - changed it back now